engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByWindow.java

```diff
@@ -231,7 +231,7 @@ abstract void computeAffectedRowsAndOperators(final UpdateByWindowBucketContext
     void assignInputSources(final UpdateByWindowBucketContext context, final ColumnSource<?>[] inputSources) {
         context.inputSources = inputSources;
         context.inputSourceGetContexts = new ChunkSource.GetContext[inputSources.length];
-        context.inputSourceChunks = new WritableChunk[inputSources.length];
+        context.inputSourceChunks = new Chunk[inputSources.length];
```

Comment:
This did not need to be `WritableChunk`, corrected.

Reply:
This was a serendipitous finding while writing the test.  `ObjectSparseArraySource.getChunk()` will return `ObjectChunk.getEmptyChunk()` when passed `Rowset.EMPTY`.  This is non-writable and will fail the array store auto-cast.